README.md

```diff
@@ -52,7 +52,9 @@ $ make codegen
 ```
 
 However, there are some deviations from the other SDKs, in which a manual code change is needed:
-- user agent parsing does not match for the Mac OS X operating system
+- `geo.py` needs to be updated according to [this commit](https://github.com/adobe/target-python-sdk/pull/34/commits/05c7bcaf9d3946e9b5a6eea719b667449e2e09fd#diff-4a8ee26e9272c52959b34df9a7763e21dcaf2d1d231fd2070dab039f63c1676e)
+- this SDK does not currently support the telemetry feature, so `TEST_SUITE_TELEMETRY.json` needs to be modified according to [this commit](https://github.com/adobe/target-python-sdk/pull/34/commits/05c7bcaf9d3946e9b5a6eea719b667449e2e09fd#diff-a328ccff9f9446689c70bdc5cb120d462646a3863f62273cf8522e6f71ac0e8e)
```

Comment:
The second bullet point feels like it might fit better under your newly added "Integration Test Generation" section.